### PR TITLE
Improve TPM2 Event Logging

### PR DESCRIPTION
**Pull request by sean-m-mcginn**
These changes include the following:

- Log config data TPM2 event as the correct type
- Log secure boot policy TPM2 event as the correct type
- Improve logs in SBL code relating to TPM2

### CODE REVIEW

File: BootloaderCommonPkg/Library/TpmLib/Tpm2Capability.c [hunk 1]
```diff
@@ -254,6 +254,11 @@ Tpm2GetCapabilitySupportedAndActivePcrs (
           *ActivePcrBanks |= HASH_ALG_SM3_256;
         }
         break;
+
+      default:
+        DEBUG ((DEBUG_VERBOSE, "GetSupportedAndActivePcrs - Unsupported bank 0x%04x.\n", Pcrs.pcrSelections[Index].hash));
+        continue;
+        break;
```
Comment (gdong1) [inline BootloaderCommonPkg/Library/TpmLib/Tpm2Capability.c:261]: I think both continue and break are not needed.

File: BootloaderCommonPkg/Library/TpmLib/TpmLib.c [hunk 2]
```diff
@@ -822,7 +822,7 @@ TpmExtendSecureBootPolicy (
 
   Data = FeaturePcdGet (PcdVerifiedBootEnabled);
 
-  Status = TpmHashAndExtendPcrEventLog (7, &Data, sizeof (Data), EV_EFI_VARIABLE_DRIVER_CONFIG, sizeof ("SecureBootPolicy"), (UINT8*)"SecureBootPolicy");
+  Status = TpmHashAndExtendPcrEventLog (7, &Data, sizeof (Data), EV_PLATFORM_CONFIG_FLAGS, sizeof ("SecureBootPolicy"), (UINT8*)"SecureBootPolicy");
```
Comment (lsubash) [inline BootloaderCommonPkg/Library/TpmLib/TpmLib.c:825]: EV_PLATFORM_CONFIG_FLAGS will be used in PCR[1] only.
This is secure boot config enable/disable one.
We can check EV_EFI_VARIABLE_AUTHORITY.

Comment (lsubash) [reply BootloaderCommonPkg/Library/TpmLib/TpmLib.c]: as discussed, we can can go with EV_PLATFORM_CONFIG_FLAGS and PCR[7].

**Review by lsubash — APPROVED** [review]
looks good

**Review by gdong1 — APPROVED** [review]

**Pull request merged**